src/Leanc.lean

```diff
@@ -0,0 +1,50 @@
+def main (args : List String) : IO UInt32 := do
+  if args.isEmpty then
+    IO.println "Lean C compiler
+
+A simple wrapper around a C compiler. Defaults to `@LEANC_CC@`,
```

Comment:
This is cool, are you moving bash shell scripts to lean?  That's awesome, and will make one day building on windows easier right?  Do we still need nix if we use Lean like this?  Is this what Lake is ?

Reply:
We never *needed* Nix :) . It's always been an alternative setup. But this + #659 + #683 will mean that we have a self-contained build system (for Lean packages, not Lean itself) without any GNU dependencies, yes.